test/step-function.localstack.test.ts

```diff
@@ -1460,6 +1460,7 @@ localstackTestSuite("sfnStack", (testResource, _stack, _app) => {
           a,
           bb: { value: b, [`${"a"}${"b"}`]: r },
           c = "what",
+          m = c,
           arr: [d, , e, ...arrRest],
           arr2: [f = "sir"],
           value,
```

Comment:
do we need tests for numeric property names? Or are they impossible in ASL?

```ts
{1: a}
```

Reply:
Latest wins

![image](https://user-images.githubusercontent.com/289213/182210343-ecf78491-4fce-4916-9df8-1ebb0c0fd7ac.png)